packages/frontend-usagers/src/pages/demande-sejour/liste.vue

```diff
@@ -297,27 +317,74 @@ const statutOptions = [
   { label: "TERMINEE", value: "TERMINEE" },
 ];
 
+const router = useRouter();
+
+const removeAttribute = (object, key) =>
+  (({ [key]: rest, ...query }) => ({ [key]: rest, query }))(object);
+
 const onUpdateId = (id) => {
+  if (id.length) {
+    router.replace({
+      query: { ...route.query, declarationId: id.join(",") },
+    });
+  } else {
+    const { query } = removeAttribute(route.query, "declarationId");
+    router.replace({ query });
```

Comment:
Ici, tu peux utilise rnavigateTO je pense , par exemple :
```
await navigateTo({
  path: '/search',
  query: {
    page: 1,
    sort: 'asc'
  }
})
```
https://nuxt.com/docs/api/utils/navigate-to